modules/homePage/component/v2/FourthJumbotron/FourthJumbotronComponent.tsx

```diff
@@ -1,7 +1,7 @@
 import ButtonPrimary from "../../../../../uikit/buttonV2/ButtonPrimary";
 
 function ForthJumobtronBox({ children, className }: { className?: string; children?: React.ReactNode }) {
```

Comment:
Btw the component name for `box` and `title` are typo sir, should've been `FourthJumbotronBox` and `FourthJumbotronTitle` hehe